pages/common/gofmt.md

```diff
@@ -0,0 +1,19 @@
+# gofmt
+
+> Gofmt formats Go programs. It uses tabs for indentation and blanks for alignment.
```

Comment:
How about `Tool for formatting Go source code` ? It matches with the `go` page. We can remove the next sentence. The exact details of the formatting specification is not required here.